extensions/autodoc_field/__init__.py

```diff
@@ -0,0 +1,115 @@
+from sphinx.ext.autodoc import AttributeDocumenter, ClassDocumenter
+from typing import Sequence
+
+import odoo
+import datetime
+
+
+class OdooClassDocumenter(ClassDocumenter):
```

Comment:
The printing of the class name is really gross:
![Screenshot 2021-12-07 at 13-16-17 Account Tag — Odoo 15 0 documentation](https://user-images.githubusercontent.com/7571158/145027493-81bc4e18-dc12-48d4-80e3-c50c40ae1d4d.png)

And it doesn't seem super useful given users don't interact with it at all (whether to use or extend the model).

Maybe derive `class` as well, change the label to `model`, and that way we don't need to override `directivetype` because the objtype can match it?

Reply:
I'm not sure I understand what you mean

Reply:
Which part?

* showing the classname is not useful since odoo developers don't interact with class(names) directly
* the class name is misrendered (cf boxes separating module and class names)
* the `class` moniker (leading keyword) is confusing as in odoo terms this is a model
* both could be resolved by overriding the class directive with one using the `model` moniker and having the auto-documenter give it the model name as "class name".

Reply:
I changed the label to "model", and I also changed the class module/name, but it didn't look good with the link to the sources it makes sense to have the full class name.
It is also good to see in an eye what Odoo module we are referring to.